packages/rust/tangram.tg

```diff
@@ -323,6 +327,61 @@ export let build = async (...args: tg.Args<Arg>): Promise<tg.Artifact> => {
 	});
 };
 
+export type VendoredSourcesArg = {
+	source: tg.Artifact;
+	useCargoVendor?: boolean;
+};
+
+let vendoredSources = async (arg: VendoredSourcesArg): Promise<tg.Template> => {
+	let { source, useCargoVendor = false } = arg;
+	if (useCargoVendor) {
+		// Run cargo vendor
+		let certFile = tg`${std.caCertificates()}/cacert.pem`;
+		let vendorScript = tg`
+			SOURCE="$(realpath ${source})"
+			mkdir -p "$OUTPUT/tg_vendor_dir"
+			cd "$OUTPUT"
+			cargo vendor --versioned-dirs --locked --manifest-path $SOURCE/Cargo.toml tg_vendor_dir > "$OUTPUT/config"
+		`;
+		let rustArtifact = rust();
+		let sdk = std.sdk();
+		let result = await std.build(tg``, {
+			env: std.env(sdk, {
```

Comment:
Minor nitpick, but `std.env` is superfluous here.  You can just pass an array of your args: `[sdk, { ...}]`